dbt/include/athena/macros/materializations/models/table/table.sql

```diff
@@ -7,10 +7,14 @@
   {%- set lf_grants = config.get('lf_grants') -%}
 
   {%- set table_type = config.get('table_type', default='hive') | lower -%}
+  {%- set temp_schema = config.get('temp_schema') -%}
```

Comment:
do we need to add same config for other materialisations: incremental, seed, snapshot ?

Reply:
Incremental was already covered.
Let's narrow down the changes of this PR, the smallest the better.
Seeds and snapshot are relevant definitely,but I recommend to cover only table materialization here.

Reply:
Yes, incrementals was already coververd in https://github.com/dbt-athena/dbt-athena/issues/613 and https://github.com/dbt-athena/dbt-athena/pull/659.
Indeed, I scoped out seed and snapshot, focusing this PR to cover only table materialization.